### PR TITLE
Adding paper on holder binding to advance reading section

### PR DESCRIPTION
**Pull request by awoie**
Adding paper on holder binding to advance reading section

### CODE REVIEW

**Review by elenachachkarova — CHANGES_REQUESTED** [review]
Thank you for your submission. I will merge your code when we add the listing.

File: advance-readings/README.md [hunk 1]
```diff
@@ -58,6 +58,11 @@ _Please add a level three header (`###`) for your paper's topic if it's not ther
...
    * #eSSIF-Lab #SSI-ecosystem #Europe
 
+### [Verifiable Credentials Holder Binding](./verifiable-credentials-holder-binding.md)
+   * by [Oliver Terbu](mailto:oliver.terbu@spruceid.com), Spruce Systems, Inc., Berlin/New York
+   * A proposal how to define a flexible and deterministic approach to verify the binding between the holder and the credential subject of the verifiable credential which is a blindspot of the W3C Verifiable Credentials Data Model 1.0 standard today.
+   * #VerifiableCredentials ##HolderBinding #2FA #Biometrics #Delegation
+
 ### ... more ...
   
 ## Alphabetical Listing
```
[the remark this answers is not in the document]
Comment (awoie) [reply advance-readings/README.md:68]: @elenachachkarova thanks for reviewing my PR. Yes, I'd prefer if you could just modify and then merge. Thanks!

[the remark this answers is not in the document]
Comment (awoie) [reply advance-readings/README.md:68]: I believe I included a link to the paper in the README.md already.

[the remark this answers is not in the document]
Comment (awoie) [reply advance-readings/README.md:68]: also link added to the alphabetical listing section

[the remark this answers is not in the document]
Comment (elenachachkarova) [reply advance-readings/README.md:68]: @awoie Thank you for making the changes. We will send yo an email with the promo code.

[the remark this answers is not in the document]
Comment (awoie) [reply advance-readings/README.md:68]: @elenachachkarova i haven't received the promo code yet. can you please send it to oliver.terbu@spruceid.com. thanks!

**Pull request merged**